### PR TITLE
Laravel 9.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 9.x.

**Before merging**, you need to:

- Checkout the `l9-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:warning: The latest version of Laravel has a dependency for `phpunit/phpunit` of `^9.3`. If you would like to also upgrade your tests, you may run the [PHPUnit 9 Shift](https://laravelshift.com/upgrade-phpunit-9) for free.

**Comment by laravel-shift** [comment]
:warning: Shift detected a dependency for the `laravel/framework`. Unless you have a special requirement for depending on the entire Laravel framework, it's recommended to add dependencies for the specific `illuminate` packages instead.

**Comment by laravel-shift** [comment]
:warning: Shift detected GitHub Actions which run jobs using a version `matrix`. Shift attempted to [update your configuration](https://github.com/vyuldashev/laravel-openapi/pull/60/commits/74d57207aab92dd78da17dcc011902cef16bbc12?w=1) for Laravel 9. However, you should review these changes to ensure the desired combination of versions are built for your package.

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/laravel-openapi.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "vyuldashev/laravel-openapi": "dev-l9-compatibility",
    }
}
```

Finally, run: `composer update`

**Comment by zlodes** [comment]
@vyuldashev may you merge this? 

And what is the status of the project? It seems that there are a lot of Issues and PRs without any reaction of maintainers.
I can help if you need.

Thanks!

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        php: [8.0]
+        php: [8.0, 8.0]
```
Comment (jasonmccreary) [inline .github/workflows/tests.yml:16]: This should not have been duplicated. We'll get it fixed up for next time.

**Comment by vyuldashev** [comment]
Thanks for running Shift for this package. 

@zlodes I would appreciate any help with this package.

**Comment by zlodes** [comment]
@vyuldashev thanks! 
Could you please make a tag from master branch? 🙏🏻

**Comment by vyuldashev** [comment]
@zlodes Done.

**Pull request merged**